app/models/user.rb

```diff
@@ -328,6 +328,10 @@ def interests
     follows.map{|follow| follow.followable.tags.map(&:name)}.flatten.compact.uniq
   end
 
+  def first_login?
```

Comment:
There's already similar logic at  `show_welcome_screen?`, couldn't we just prevent duplicating code?